modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -63,12 +63,12 @@ api.<cluster_name>.<base_domain>.  IN  A  <API_FIP>
 ====
 If you do not control the DNS server, you can access the cluster by adding the cluster domain names such as the following to your `/etc/hosts` file:
 
-* `<api_floating_ip> api.<cluster_name>.<base_domain>`
-* `<api_floating_ip> grafana-openshift-monitoring.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> prometheus-k8s-openshift-monitoring.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> oauth-openshift.apps.<cluster_name>.<base_domain>`
-* `<api_floating_ip> console-openshift-console.apps.<cluster_name>.<base_domain>`
-* `api_floating_ip integrated-oauth-server-openshift-authentication.apps.<cluster_name>.<base_domain>`
+* `<application_floating_ip> api.<cluster_name>.<base_domain>`
```

Comment:
Looking at the previous PR, this first line should still be `<api_floating_ip>`.

Reply:
my bad. Addressed

Reply:
It looks like this matches the terminology in https://github.com/openshift/openshift-docs/commit/392e1350b62bb1d689bbe3799dae79130f3550d9, so I'm going to merge it. Thank you!